conference/src/main/resources/application.properties

```diff
@@ -1 +1,6 @@
-
+spring.jpa.database=mysql
+spring.jpa.hibernate.ddl-auto=update
```

Comment:
Move DB schema generation to tool like Flyway or Liquibase because it is unreliable to use generation from mapping